lib/packwerk/application_validator.rb

```diff
@@ -24,15 +24,15 @@ def initialize(config_file_path:, application_load_paths:, configuration:)
 
     def check_all
       results = [
+        check_package_manifest_syntax,
         check_autoload_path_cache,
         check_package_manifests_for_privacy,
-        check_package_manifest_syntax,
```

Comment:
This feels like the first thing we should check.

Reply:
Not sure this should be in the same PR as your new testing pattern.

Also, If we start reordering these, I think we should do

- all application integration tests first (autoload path cache, inflections, application structure)
- all package manifest checks afterwards (manifest syntax, manifest privacy, acyclic graph etc)

Reply:
Definitely not in the same PR.  This is just a throwaway POC, where I occasionally needed to make unrelated changes in order to unblock the larger demo.

I believe one of my new unit tests could not exercise what I wanted to demonstrate until I reordered the checks.